js/ui/map.js

```diff
@@ -1417,3 +1417,19 @@ function removeNode(node) {
   * @property {string} type The event type.
   * @property {string} dataType The type of data that has changed. One of `'source'`, `'style'`, or `'tile'`.
   */
+
+ /**
+ * Fired immediately after the map container has been removed with [`Map#remove`](#Map#remove).
```

Comment:
`Map#remove` doesn't remove the `container` (the HTML element that contains the map canvas & controls). I suggest changing this to

 >  Fired immediately after the map ~~container~~ has been removed with [`Map#remove`](#Map#remove)